test/test_bad/expect/import-name.yang.out

```diff
@@ -1,6 +1,5 @@
-import-name.yang:2: error: DUPLICATE_NAMESPACE
+import-name.yang:5: error: MODULE_NOT_FOUND
```

Comment:
This seems really weird.  This fix implies that the tests currently fail, but they don't.
This comment applies to more similar changes below.